### ISSUE TITLE
Bypassing lists: accelerating screen-reader fact-finding with guided tours

### ISSUE THREAD

**Issue by Wotipati** [Wotipati/Blind_Accessibility_Papers#255]
### Links
- Paper: https://dl.acm.org/doi/10.1145/2513383.2513435

<img width="388" alt="ScreenShot 2020-05-18 3 08 42" src="https://user-images.githubusercontent.com/26996041/82156253-09415300-98b5-11ea-8da9-04bb4d2416a6.png">
<img width="791" alt="ScreenShot 2020-05-18 3 09 35" src="https://user-images.githubusercontent.com/26996041/82156256-09d9e980-98b5-11ea-935d-65bed2187246.png">

### Abstract
Navigating back and forth from a list of links (index) to its target pages is common on the web, but tethers screen-reader users to unnecessary cognitive and mechanical steps. This problem worsens when indexes lack information scent: cues that enable users to select a link with confidence during fact-finding. This paper investigates how blind users who navigate the web with screen-readers can bypass a scentless index with guided tours: a much simpler browsing pattern that linearly concatenates items of a collection. In a controlled study (N=11) at the Indiana School for the Blind and Visually Impaired (ISBVI), guided tours lowered user's cognitive effort and significantly decreased time-on-task and number of pages visited when compared to an index with poor information scent. Our findings suggest that designers can supplement indexes with guided tours to benefit screen-reader users in a variety of web navigation contexts.  

- Webにあるリストを効率的に移動できるように支援するシステム
